zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -357,7 +442,11 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
 
         // 3. Call to account with empty code.
         cb.condition(
-            and::expr([not::expr(tx_is_create.expr()), no_callee_code.clone()]),
+            and::expr([
```

Comment:
I think there is mismatch with specs https://github.com/privacy-scaling-explorations/zkevm-specs/blob/master/src/zkevm_specs/evm_circuit/execution/begin_tx.py#L200
condition

Reply:
Updated.